src/modules/roc_netio/target_libuv/roc_netio/udp_receiver_port.cpp

```diff
@@ -263,5 +274,42 @@ void UDPReceiverPort::recv_cb_(uv_udp_t* handle,
     self.writer_.write(pp);
 }
 
+bool UDPReceiverPort::join_multicast_group_() {
+    char host[128];
+    address_.get_host(host, sizeof(host));
+
+    char miface[128];
+    address_.get_miface(miface, sizeof(miface));
```

Comment:
Lets add and use corresponding constant to packet::Address, something like Address::MaxStrLen.

Reply:
Done